alvr/server/cpp/platform/linux/CEncoder.cpp

```diff
@@ -232,11 +232,19 @@ void CEncoder::Run() {
         // Close enough to present
         ReportPresent(pose->targetTimestampNs);
 
+        if (access("/tmp/alvr-capture-input", F_OK) == 0) {
```

Comment:
file should be created automatically(according to zarik) and then guarded by a advanced setting

Reply:
goes for both(obviously)

Reply:
I don't understand. This code just checks if this file exists.
As explained in commit message, it is meant that you will `touch /tmp/alvr-capture-input` (aka create file) > this code will detect this file exists > capture next frame and delete that file (so it only captures one frame).

Reply:
is it possible to add this functionality in the dashboard? like with a button in the Installation tab? (it's misplaced, maybe we should add a Debug tab in the sidebar?)

Reply:
If you give me pointers how to do that, sure. Currently there doesn't seem to be a way to push a change from dashboard up to encoder thread.
`Settings.h` is read-once, it won't notice the change until restart.

Reply:
Currently we use a hacky and unelegant trick, we copy settings to a section of session.json and then the c++ code loads that into the Settings.h singleton. This is done so that every change to this section triggers a restart of steamvr to make sure settings are up to date. Check connection.rs

Reply:
You can provide the path as a settings, in command.rs you copy the path to OpenvrConfig, load the setting from Settings.cpp. To sen the command, add a button to the dashboard, let it invoke a GET request, on web_server.rs add an API path (like "/api/capture-frame"), here invoke a C++ function you defined in bindings.h.